pkgs/servers/web-apps/wordpress/wordpress-core.nix

```diff
@@ -0,0 +1,12 @@
+{ stdenv, wordpress }:
+
+with stdenv.lib;
```

Comment:
What is this needed for?

Reply:
You mean the whole file? Wordpress ships with some themes (twentytwenty currently I think, maybe also twentynineteen and twentyeightteen) and plugins (akismet, hello dolly) and these can't be removed or updated if there isn't a new wordpress version released yet. So by removing them you can independently remove or update them using the plugins and themes attribute and therefore also limit the amount of potentially vulnerable code.

Reply:
If you meant the stdenv thingy - I removed it.